tests/installation/disk_activation_zfcp.pm

```diff
@@ -0,0 +1,73 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: This test module handles ZFCP disk activation
+#          through libyui-rest-client.
+
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use YuiRestClient::Logger;
+
+sub run {
+    # in Disk Activation page press "ZFCP" button
+    my $disk_activation_selector = $testapi::distri->get_disk_activation_selector();
+    $disk_activation_selector->zfcp();
+
+    # in the ZFCP Device overview press the "Add" button
+    my $zfcp_configuration_overview = $testapi::distri->get_zfcp_configuration_overview();
+    $zfcp_configuration_overview->add();
+
+    # In the Add Device page insert hard wired channel id and press next
+    my $zfcp_add_disk = $testapi::distri->get_zfcp_add_disk();
+    $zfcp_add_disk->set_channel("0.0.fa00");
```

Comment:
We have tried to hide the `next` all that we can, in this case you can just have a `configure` method with named params, i.e.: `->configure(channel => '0.0.fa00')` (btw, there is not need to double quotes when there is not interpolation, but if you prefer like that is ok which does the next inside.